packages/@vue/cli-service/generator/hotfix-npm6only.js

```diff
@@ -0,0 +1,8 @@
+// this file will be renamed to hotfix.js if the package is installed by npm 6
+module.exports = (api) => {
+  api.extendPackage({
```

Comment:
You can check the package manager version here and avoid the postinstall

Reply:
The generator is controlled by `@vue/cli` itself, so npm is not involved during the execution, therefore the package manager version is not available.

I don't feel like adding an ad hoc API just for this fix, so I chose to keep the postinstall script.